app/views/companies/show.html.erb

```diff
@@ -3,6 +3,42 @@
   <%= link_to 'Stock', companies_path, class: "breadcrumb-item small text-decoration-none" %>
   <span class="breadcrumb-item active small"><%= @company.name %></span>
 </nav>
-<h2>Company name: <%= @company.name %></h2><br />
-<h4>Company symbol: <%= @company.isin %></h4><br />
-<h5 class="text-muted">Id in database: <%= @company.id %></h5>
+
+<div class="container">
+  <div class="container">
+    <h4><%= @company.name %></h4>
+    <p class='small text-muted'><%= @company.belong_to %></p>
+  </div>
+  <div class="col m-2">
+    <div class="row">
+      <div class="col">
+        <p>P/BV</p>
+        <p>P/E</p>
+        <p>Capitalization</p>
+        <p>Book value</p>
+        <p>Shares</p>
+        <p>Stock</p><br />
+      </div>
+      <div class="col">
+        <p><%= @company.pbv ? @company.pbv.ceil(2) : 'n/a' %></p>
+        <p><%= @company.pe ? @company.pe.ceil(2) : 'n/a' %></p>
```

Comment:
You could use presenter pattern to get rid of such logic in the view. To keep the view without any logic and focused purely on rendering.

Reply:
This is good solution

Reply:
But I'm not sure which localization chose for the file, I mean `app/presenters` , `lib/presenters` , `lib/companies` , 
what do you think?

Reply:
`app/presenters` is fine. I think this is part of the "web" layer of the app as you are going to cover presenting objects for HTML rendering purpose.